pkgs/development/mobile/androidenv/generated/packages.nix

```
@@ -1132,145 +1135,96 @@
     };
   };
   
-  "emulator"."30.0.3".linux = {
+  "emulator"."30.0.16".macosx = {
```

I think this is a mistake? should be 
```suggestion
  "emulator"."30.0.16".linux = {
```